### PR TITLE
Add and correct Hebrew translations

### PR DESCRIPTION
**Pull request by amire80**

### CODE REVIEW

File: src/partials/languages/he.html [hunk 1]
```diff
@@ -24,22 +24,42 @@
...
+  "header.title": "כותרת עליונה",
   "meta.comments": "הערות",
-  "meta.source": "מָקוֹר",
+  "meta.source": "מקור",
+  "nav.title": "ניווט",
+  "search.config.lang": "he",
   "search.config.pipeline": " ",
-  "search.placeholder": "לחפש",
-  "search.result.placeholder": "הקלד כדי להתחיל לחפש",
+  "search.config.separator": "[\\s\\-]+",
```
Comment (squidfunk) [inline src/partials/languages/he.html:43]: The separator is only necessary to provide if it deviates from the standard one defined for `en` which it doesn't. Please remove this line. I'm going to improve the documentation for providing and updating translations in the near future.

**Review by squidfunk — CHANGES_REQUESTED** [review]

File: src/partials/languages/he.html (hunk 1 above)
Comment (amire80) [reply src/partials/languages/he.html]: Thanks for the comment. Done. I actually had a suspicion about it.

File: src/partials/languages/he.html [hunk 2]
```diff
@@ -24,22 +24,41 @@
...
+  "edit.link.title": "עריכת הדף הזה",
+  "footer.previous": "הקודם",
+  "footer.next": "הבא",
+  "footer.title": "כותרת תחתונה",
+  "header.title": "כותרת עליונה",
   "meta.comments": "הערות",
-  "meta.source": "מָקוֹר",
+  "meta.source": "מקור",
+  "nav.title": "ניווט",
+  "search.config.lang": "he",
```
Comment (squidfunk) [inline src/partials/languages/he.html:41]: Oh, also, `he` is not available as a search language. Please remove. Search should work without this separator.

**Review by squidfunk — CHANGES_REQUESTED** [review]

File: src/partials/languages/he.html (hunk 2 above)
Comment (amire80) [reply src/partials/languages/he.html]: Done.

**Comment by squidfunk** [comment]
Thanks! LGTM.

**Pull request merged**